roles/multi-tenancy/tasks/multi-tenant-onboard-offboard.yaml

```diff
@@ -135,9 +133,9 @@
   ansible.builtin.shell: |
     kubectl --kubeconfig {{ KUBECONFIG }} get pods -n {{ NAMESPACE }} --sort-by=.metadata.creationTimestamp | tac | grep sas-tenant-onboard | awk '{print $3}'
   register: pod_status
-  until: pod_status.stdout_lines[0] == "Completed"
-  retries: "{{ tenant_count|int }}"
-  delay: 75
+  until: pod_status.stdout_lines[0] == "Completed" or pod_status.stdout_lines[0] not in pod_fail_list
+  retries: 60
+  delay: 60
```

Comment:
both of these value should be variables that are set in the `defaults` section to these values. This allows us to surface the items if needed and/or make changed with just variable changes not hard coded values.

Reply:
Added new variable in the latest change.